DXRando/DeusEx/Classes/JuanLebedev.uc

```diff
@@ -0,0 +1,12 @@
+class DXRJuanLebedev injects JuanLebedev;
```

Comment:
I don't think we should do this like this, it won't work with other mods and it doesn't match our other methods of tracking deaths for bingo, we should be checking for his death in DXREvents